pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -188,7 +188,7 @@ func newGCECloud(config io.Reader) (*GCECloud, error) {
 	if err != nil {
 		return nil, err
 	}
-	networkURL := gceNetworkURL(apiEndpoint, projectID, networkName)
+	networkURL := gceNetworkURL("", projectID, networkName)
```

Comment:
There are a few other calls to `gceNetworkURL` - why do they get apiEndpiont instead of ""?